### DISCUSSION TITLE
Invert z-axis option wanted.

### DISCUSSION THREAD

**Discussion by maghauke** [foxglove/community#442] [Q&A]
I want to use the 3D panel, but my /tf are up-side down, so I want to use the "invert z-axis" property as in rviz. Is this possible to enable?

**Comment by jhurliman** [comment]
The ability to edit transforms is in progress, so you will be able to apply a 180 degree rotation to the correct axis when that feature lands.

**Comment by jhurliman** [comment]
The transform editing PR is up, but it can't achieve the same behavior as inverting the z-axis since that would require adding a new transform above the current root transform in a scene. I'll leave this open as a feature request to be implemented separately.
